server/conf/routes

```diff
@@ -65,21 +65,21 @@ POST    /admin/questions/:id/restore controllers.admin.AdminQuestionController.r
 POST    /admin/questions/:id/archive controllers.admin.AdminQuestionController.archive(request: Request, id: Long)
 
 # Routes for editing question localizations
-GET     /admin/questions/:id/translations/:locale/edit  controllers.admin.AdminQuestionTranslationsController.edit(request: Request, id: Long, locale: String)
-POST    /admin/questions/:id/translations/:locale       controllers.admin.AdminQuestionTranslationsController.update(request: Request, id: Long, locale: String)
+GET     /admin/questions/:id/translations/:locale/edit                    controllers.admin.AdminQuestionTranslationsController.edit(request: Request, id: Long, locale: String)
```

Comment:
Blocking: Looks like some whitsespace changes were introduced here unnecessarily? Mind reverting?

If I'm reading the diff incorrectly, please ignore.

Reply:
![image](https://user-images.githubusercontent.com/101214724/181411803-a1c0dd8f-cd2e-4f57-adce-06329a70b831.png)
Bion asked me to fix the alignment and I did it.. This is very odd why it is rendering like this.